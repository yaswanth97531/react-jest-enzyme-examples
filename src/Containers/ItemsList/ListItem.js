import React from 'react';
import PropTypes from 'prop-types';

/*
Render a single item
@param {Object} props
 */

 function ItemList(props) {
   const { item } = props;
   return <li id='item'>{item}</li>
 }

 ItemList.propTypes = {
   item: PropTypes.string,
 };

 export default ItemList;