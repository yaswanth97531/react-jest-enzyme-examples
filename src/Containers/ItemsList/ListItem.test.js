import ListItem from './ListItem';

let wrapper;

beforeEach(() => {
  wrapper = mount( <ListItem / > );
});

describe('<App /> rendering', () => {
  it('should render correctly', () => {
    const tree = renderer.create( <ListItem / > ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
describe('<App /> interactions', () => {});
describe('<App /> lifecycle method invocations', () => {});