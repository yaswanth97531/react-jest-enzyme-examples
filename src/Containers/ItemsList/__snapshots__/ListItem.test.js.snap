// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> rendering should render correctly 1`] = `
<li
  id="item"
/>
`;
